doc/end_to_end_pipeline.rst

```diff
@@ -6,26 +6,26 @@ End-to-end predictive models
 
 .. currentmodule:: skrub
 
-.. _table_vectorizer:
+.. _tabular_learner:
 
-Turning a dataframe into a numerical feature matrix
----------------------------------------------------
+Create baseline predictive models on heterogeneous dataset
+----------------------------------------------------------
 
-A dataframe can contain columns of all kind of types. A good numerical
-representation of these columns help analytics and statistical learning.
+Crafting a machine-learning pipeline is rather a daunting task. Choosing the ending
+learner of such pipeline is usually the easiest part. However, it imposes constraints
+regarding the preprocessing steps that are are required ahead of the learner.
+Programmatically defining these steps is the part that requires the most expertise and
+that is cumbersome to write.
 
-The :class:`TableVectorizer` gives a turn-key solution by applying
-different data-specific encoders to the different columns. It makes reasonable
-heuristic choices that are not necessarily optimal since it is not aware of the learner
-used for the machine learning task). However, it already provides a typically very good
-baseline.
+The function :func:`tabular_learner` provides a factory function that given a
+scikit-learn estimator, returns a pipeline that combines this estimator with the
+appropriate preprocessing steps. Those steps corresponds to a :class:`TableVectorizer`
+that is in charge of dealing with heterogeneous data and depending on the capabilities
+of the final estimator, a missing value imputer.
```

Comment:
and/or a standard scaler?

Reply:
Right, it was added since I started this PR :)